bot/kodiak/queue.py

```diff
@@ -546,7 +555,8 @@ def _start_worker(
             sentry_sdk.capture_exception(exception)
         log.info("creating task for queue")
         # create new task for queue
-        self.worker_tasks[key] = (asyncio.create_task(fut), kind)
+        # TODO: before merge fix this
+        self.worker_tasks[key] = (asyncio.create_task(fut), kind, "TODO-id")
```

Comment:
we need the task to have access to the worker heartbeat data structure which it needs to update periodically with its status

Reply:
not sure how to do that yet, may need to restructure some things